src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/InstanceHelper.cs

```diff
@@ -70,5 +80,38 @@ public static List<SblInstanceEvent> ConvertToSBLInstanceEvent(List<InstanceEven
 
             return simpleEvents;
         }
+
+        /// <summary>
+        /// Gets the equivalent sbl status for a given instance status
+        /// </summary>
+        /// <param name="instance">the instance</param>
+        /// <returns>status</returns>
+        public static string GetSBLStatusForCurrentTask(Instance instance)
+        {
+            if (instance.Process != null)
+            {
+                string currentTask = instance.Process.CurrentTask?.ElementId;
+                if (currentTask != null && currentTask.Equals(Task1))
+                {
+                    return "FormFilling";
+                }
+                else if (currentTask != null && currentTask.Equals(Task2))
```

Comment:
Since task2 doesn't exist you should use: if (instance.Process.CurrentTask == null && instance.Process.Ended != null) { return "Submit"; }

Reply:
But doenst that mean archived?